pkg/helm/charts/tests/centralservices/testdata/all-values-explicit.yaml

```
@@ -45,6 +45,24 @@ scanner:
   dbServiceTLS:
     cert: "scanner-db tls cert pem"
     key: "scanner-db tls key pem"
+scannerV4:
+  disable: false
+  indexer:
+    disable: false
+    serviceTLS:
+      cert: "scanner tls cert pem"
+      key: "scanner tls key pem"
+  matcher:
+    disable: false
+    serviceTLS:
+      cert: "scanner tls cert pem"
+      key: "scanner tls key pem"
+  db:
+    serviceTLS:
+      cert: "scanner tls cert pem"
+      key: "scanner tls key pem"
+    password:
+      value: "db password"
```

I think those TLS secret should be references to `type: TLS` kubernetes secrets, not plaintext. And the structure closely aligned with the `tls.Config{}` interface. It would allow us to later configure different TLS configurations when/if we implement a shared scanner on ACSCS.

```suggestion
scannerV4:
  disable: false
  indexer:
    disable: false
    tls:
      certificates: 
        - secretName: ...
  matcher:
    disable: false
    tls:
      certificates:
        - secretName: ...
  db:
    tls:
      certificates: 
        - secretName: ...
    password:
      secretName: ...
```

---

Hey, yes this was probably the wrong file. 
My only concern is that certs/secrets should never be specified plaintext in manifests. 
They should only be "injected" (referenced). I think that otherwise, this would have consequences in how the operator / central / secured cluster logic will actually be able to perform the automatic TLS rotation.